SSA/Projects/Tensor1D/Tensor1D.lean

```diff
@@ -294,142 +294,74 @@ theorem Tensor1d.tile [Inhabited α] (t : Tensor1d α) (SIZE : 4 ∣ t.size) (f
     sorry
 }
 
-
-
-
-
-
-
 namespace ArithScfLinalg
 
-open Val
 
-inductive Op
+
+inductive ExOp
```

Comment:
why the rename?

Reply:
To be consistent with the others in InstCombine

Reply:
Not sure I understand, `InstCombine` doesn't call them `ExOp`, it calls them `Op` (and similarly for `Ty`)

Reply:
I'd argue for sticking with `Op` as well.
I think @bollu is referring to the example dialects in the `Framework.lean` (formerly `IntrinsicAsymptotics`) file, which do use  the`Ex-` prefix to make it explicit they are examples, but those should arguably be put in their own file anyway